usaspending_api/core/validator/tinyshield.py

```diff
@@ -225,11 +264,12 @@ def enforce_rules(self):
                 self.recurse_append(struct, self.data, self.apply_rule(item))
 
     def apply_rule(self, rule):
+        _return = None
         if rule.get('allow_nulls', False) and rule['value'] is None:
-            return rule['value']
-        elif rule['type'] not in ('array', 'object'):
+            _return = rule['value']
+        elif rule['type'] not in ('array', 'object', 'any'):
             if rule['type'] in VALIDATORS:
-                return VALIDATORS[rule['type']]['func'](rule)
+                _return = VALIDATORS[rule['type']]['func'](rule)
             else:
                 raise Exception('Invalid Type {} in rule'.format(rule['type']))
```

Comment:
Perhaps you should pull the exception code into a function so the messages are uniform with the nicer output you created in lines 321-325

Reply:
ignore